app/models/post.rb

```diff
@@ -1,3 +1,7 @@
 class Post < ApplicationRecord
-  belongs_to :author_id
+  belongs_to :user
```

Comment:
- [ ] I think you forget to add relation between post table and comments and likes since in the ERD it shown that each post has many comments and many likes so please add these relations to your Post model 🤩✅✨🔥💯